README.md

```diff
@@ -1,46 +1,62 @@
-# Getting Started with Create React App and Redux
+# Calculator
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+This app implements a simple calculator. This is an exercise in design and development, using React, Redux and CSS in JS.
 
-## Available Scripts
+## Features
 
-In the project directory, you can run:
+The app contains three pages:
 
-### `npm start`
+### Calculator (home page)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+Composed of a thin strip (aprox. 280px) with variable height. It is composed of different section.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+#### Keys (tabbable)
 
-### `npm test`
+The numbers and opperations dispatches write commands to the store. The **clear** key dispatches a delete all command, and the **back** key dispatches a delete command for the last digit. The **save** key register dispatches a save command with the current expression, so it can appear in the Scope and in the History pages.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+The Keys group is accessible by tabbing. In fact, they are divided into two distinct tabbable groups, one containing 0-9, period and basic arythmetic operations, and another containing the other keys. The ARROWS can be used to select a specific key, including moveing from one group to the other (and even to other components, as scope and navigation). Once a key is focused, it can be activated by the SPACEBAR or ENTER keys. The ESC key activates the clear key.
 
-### `npm run build`
+#### Visor (non-tabbable)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+It displays the current expression, fed from the store, and the calculated sum. Calculations are made in real time. It should clearly display (through colors and visually) when the expression is malformed.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+The visor is not interactive and non tabbable. Clicking over it **does not allow** the user to set a cursor position for editing the expression. That would create a complexity that is not in the main focus of this project.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+When tabbed to the number group, 5 is selected by default. When tabbing to the secondary group, save is selected by default.
```

Comment:
What if the user had moved the to another number, say 7, then pressed tab and then pressed shift tab. Should it go back to 5 or should it remember it was on 7?

Reply:
I don't think remembering the last the last button will be very valuable, as I don't think the user himself would. Going to a default key would could create a habit and make it easier. This issue could benefit from some usability testing, but giving the limited resources, I'd vote for using default buttons.